docs/reference-docs/auth-backend-and-frontend.md

```diff
@@ -0,0 +1,228 @@
+# Authentication and authorization
+
+WFO can be run with or without authentication. With authentication turned on authorization logic can be provided that uses - for example - user privileges to allow further access to resources.
+
+Authentication is configured using ENV variables. The frontend and backend have their own set of ENV variables and logic to be implemented to run auth(n/z).
+
+## Definitions
+
+A **frontend application** refers to a web frontend based on the frontend example ui repository: [frontend repo][1]
+A **backend application** refers to an application build using the orchestrator core as a base: [backend repo][2]
+
+## Without authentication
+
+Without authentication WFO allows all users access to all resources.
```

Comment:
without authentication there are no users.

Not sure if you mean authorisation here?

Reply:
Changed it to visitors